### PR TITLE
create database entity for TrayReturns

### PR DESCRIPTION
**Pull request by ZD1wang**
# Description
create tray_return api, model and test

## Type of change
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?
-  test created in test_main.py file
-  manual api testing through postman

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I will rebase before merging

### CODE REVIEW

**Comment by atykwonderland** [comment]
ur code looks good! 
but the three separate tests for the various tray components all have the same body content so it's a little hard to tell if its actually working properly without going into ur code every time.. i would just change that for later crud tests to make it easier to differentiate

File: backend/tests/api/test_main.py [hunk 1]
```diff
@@ -97,21 +97,30 @@ def test_remove_food(self):
...
     def test_vendors_tray(self):
-        # Add tray_collect first
-        payload = {
+        # Add tray_collect
+        tray_collect_payload = {
             "pickup_time": "2020-10-08 11:11:51.291273",
             "vendor_id": 1,
         }
-        response = self.app.post("/add_tray_collect", json=payload)
+        self.app.post("/add_tray_collect", json=tray_collect_payload)
```
Comment (atykwonderland) [inline backend/tests/api/test_main.py:106]: ur code looks good!
but the three separate tests for the various tray components all have the same body content so it's a little hard to tell if its actually working properly without going into ur code every time.. i would just change that for later crud tests to make it easier to differentiate

Comment (ZD1wang) [reply backend/tests/api/test_main.py]: done!

File: backend/tests/api/test_main.py [hunk 2]
```diff
@@ -297,3 +318,19 @@ def test_create_tray_collect(self):
...
+        vendor = {
+            "name": "hitata",
+            "address": "defeat road",
+            "city": "zootopia",
+        }
+        response = self.app.post("/add_vendor", json=vendor)
+
+        payload = {
+            "return_time": "2020-10-08 11:11:51.291273",
+            "vendor_id": 1,
```
Comment (taerim25) [inline backend/tests/api/test_main.py:333]: Instead of hard coding vendor_id, I think it's better to use id from the response.

Comment (ZD1wang) [reply backend/tests/api/test_main.py]: done!

**Pull request merged**